src/core/attribute.d

```diff
@@ -136,7 +136,7 @@ version (UdaGNUAbiTag) struct gnuAbiTag
 {
     string[] tags;
 
-    this(string[] tags...)
```

Comment:
Why are these mutually exclusive?

Reply:
Not sure what you mean.

Reply:
Why does `this(scope string[] tags...)` not work?

Reply:
Still not sure what you mean. `scope` is applied automatically to ... parameters. The replacement for `T[]...` is `scope T[]`.